### DISCUSSION TITLE
I plan to export a function in the package, but after packaging, no export content

### DISCUSSION THREAD

**Discussion by dddssw** [webpack/webpack#18789] [Q&A]
I plan to develop an npm package to provide a method for my vscode plugin to use. After that, I happily use webpack to package it, but the packaged file does not export anything. I checked it for a long time and did not find the problem. There must be something I don't know.

webpack.config.js
``

```
import path from "path";

// Webpack 配置
const config = {
  entry: "./src/index.js", 
  output: {
    filename: "bundle.js", 
    path: path.resolve(process.cwd(), "dist"), 
    library: "exportInfo", 
    libraryTarget: "umd",
    globalObject: "this",
    libraryExport: "default",
  },
  mode: "production",
};

export default config; 

```
index.js
```
import { parse } from "@babel/parser";
import traverse from "@babel/traverse";
export const test = "Test";
export default function(code, defaultName) {
............
}

```
bundle.js does not export anything

**Comment by alexander-akait** [comment]
Why you have `libraryTarget: "umd"` and `libraryExport: "default"`?
